Design.md

```diff
@@ -238,6 +254,12 @@ This API query S3 credentials of the  user using a `filter` parameter.
 ### List S3 Credentials
 This API list S3 credentials of the  user.
 1. `list-access-keys` api will be called using assumed role credentials.
+2. Retrieve the `secret-keys` from Redis cache with key formatted as `<Username>__<AccessKeyID>`.
+3. Decrypt the `secret-keys` using AES and add them to the response.
```

Comment:
Same here, weren't we using base 64? Or was it changed on Friday to using AES?

Reply:
Sai described to me on Friday as using AES. 

Given the description above using `<Username>__<AccessKeyID>` as the method to store/retrieve the secret key in Redis cache would not be secure enough with base64 encoding. I'd presume AES change was to prevent access to Redis cache allowing associating access key and a decoded secret key.

Reply:
As mentioned above,

> This is changed to AES after discussing with Vianney over Slack. We decided that AES is more secure when stored on Redis than just base64 encoding. AES private key will be provided on `application.properties` for OSIS and should be common for all nodes running OSIS.

Reply:
Cool, Just wanted to make sure this is what we are intending to do. I was on PTO so this was new to me.
Makes sense on the security side.

Also we will be using hash key to avoid root namespace: https://github.com/scality/vmware-ose-scality/pull/45#discussion_r632161118